aptos-move/aptos-gas/src/transaction/storage.rs

```diff
@@ -265,6 +267,46 @@ impl ChangeSetConfigs {
     }
 }
 
+#[inline]
+fn size_error() -> VMStatus {
+    VMStatus::Error(StatusCode::STORAGE_ERROR, None)
+}
+
+impl SizeChecker for ChangeSetConfigs {
```

Comment:
@msmouse What do you think about having checks per different types? I will be adding a new generic write type and  hopefully with some traits storage backend can also use it (is it needed there though or it's just VM side?)